oecologia-australis.csl

```diff
@@ -142,38 +138,38 @@
         </choose>
         <text variable="number-of-pages" prefix="p. " suffix="."/>
         <group prefix=" ">
-          <text term="retrieved" text-case="capitalize-first" suffix=" "/>
-          <text term="from" suffix=" "/>
-          <text variable="URL"/>
+          <choose>
+            <if type="chapter" match="none">
+              <text term="retrieved" text-case="capitalize-first" suffix=" "/>
+              <text term="from" suffix=" "/>
+              <text variable="URL"/>
+            </if>
+          </choose>
         </group>
       </if>
+      <else-if type="legislation webpage" match="any">
+        <text value="Retrieved in" prefix="("/>
+        <date form="text" variable="accessed" prefix=" " suffix=", ">
+          <date-part name="day" form="ordinal"/>
+          <date-part name="month"/>
+          <date-part name="year"/>
+        </date>
+        <text value="from " prefix=" "/>
+        <text variable="URL" strip-periods="false" quotes="false" font-variant="normal" suffix=")."/>
```

Comment:
@gudryan, it's very fragile to split parenthesis over multiple elements (`<text value="Retrieved in" prefix="("/>` and `<text variable="URL" ... suffix=")."/>`). In this case, it's much better to put everything here in a group that use parentheses:

```xml
<group prefix="(" suffix=")">
   ...
</group>
```